project/profiles/capacity4more/behat/features/bootstrap/FeatureContext/GroupMerge.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace FeatureContext;
+
+/**
+ * @file
+ * Context methods about Groups merge.
+ */
+
+/**
+ * Definitions of tests for group merge.
+ *
+ * @package FeatureContext
+ */
+trait GroupMerge {
+
+  /**
+   * Helper to get the node based on the title.
+   *
+   * @param string $title
+   *   The group title.
+   *
+   * @return int
+   *   The node id or NULL.
+   *
+   * @throws Exception
+   *   Throws exception if group title not found.
+   */
+  private function getNodeIdByTitle($title) {
+    $query = new \entityFieldQuery();
```

Comment:
Don't we have a similar function already?

Reply:
The function is similar, but the existing one depends on the type of the group and loads and returns the group object which is not necessary in this case.